service/kg-search/src/main/java/eu/ebrains/kg/search/KgSearchApplication.java

```diff
@@ -34,6 +34,7 @@
 import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
 import org.springframework.security.config.http.SessionCreationPolicy;
 import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.web.client.RestTemplate;
```

Comment:
Java also has a default http client (see https://www.baeldung.com/java-9-http-client ) which could be used instead of the spring specific RestTemplate - since we are in a spring context anyhow, this is not wrong though and you can leave it as is.